src/test/end-to-end/SignIn.test.js

```diff
@@ -0,0 +1,22 @@
+// used the resource: https://www.youtube.com/watch?v=r9HdJ8P6GQI to guide e2e test to understand puppeteer syntax
+// with html file that was converted from the Login.jsx file since the react/js integration was giving me trouble.
+import puppeteer from 'puppeteer';
```

Comment:
Same feedback as on the other parts of the assignment. This is and end-to-end test. Well done, however it's not testing the actual production application. Which is fine for the assignment.